### PR TITLE
Fixing build issue in windows

### PR DESCRIPTION
**Pull request by ndthanhit**

### CODE REVIEW

File: konduit-serving-models/konduit-serving-tvm/pom.xml [hunk 1]
```diff
@@ -53,6 +53,16 @@
...
       <version>${nd4j.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.bytedeco</groupId>
+          <artifactId>dnnl-platform</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.bytedeco</groupId>
+          <artifactId>dnnl</artifactId>
+        </exclusion>
```
Comment (agibsonccc) [inline konduit-serving-models/konduit-serving-tvm/pom.xml:64]: Was this necessary? I prefer not to use exclusions unless absolutely necessary.

File: pom.xml [hunk 2]
```diff
@@ -190,6 +188,11 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-stdlib-jdk8</artifactId>
+                <version>1.4.32</version>
```
Comment (agibsonccc) [inline pom.xml:194]: Add a standardized version of this as a property following the rest of the projects.

File: pom.xml [hunk 3]
```diff
@@ -543,12 +546,26 @@
                         <groupId>org.bytedeco</groupId>
                         <artifactId>mkl-dnn-platform</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.bytedeco</groupId>
```
Comment (agibsonccc) [inline pom.xml:550]: This exclusion in..log4j? doesn't make any sense. Did you add this in the right place?

File: pom.xml [hunk 4]
```diff
@@ -543,12 +546,26 @@
...
+                        <artifactId>javacpp</artifactId>
+                    </exclusion>
                 </exclusions>
             </dependency>
             <dependency>
                 <groupId>org.bytedeco</groupId>
                 <artifactId>tensorflow-platform</artifactId>
                 <version>${tensorflow.javacpp.version}</version>
+                <exclusions>
+                    <exclusion>
```
Comment (agibsonccc) [inline pom.xml:560]: This one I'm fine with. Add comments explaining any exclusion you add though.

**Review by agibsonccc — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (ndthanhit) [reply pom.xml:550]: No it's not in log4j, you can see the line number is different.

File: konduit-serving-models/konduit-serving-tvm/pom.xml (hunk 1 above)
Comment (ndthanhit) [reply konduit-serving-models/konduit-serving-tvm/pom.xml:64]: it's necessary, otherwise the build will have error.

Comment (agibsonccc) [reply konduit-serving-models/konduit-serving-tvm/pom.xml:64]: @ndthanhit  can you add a comment or something showing the dependency trace?

File: pom.xml (hunk 3 above)
Comment (agibsonccc) [reply pom.xml:551]: Ah I see. Github's interface is weird.

**Review by agibsonccc — APPROVED** [review]

**Pull request merged**